src/Acheve.TestHost/IncludeContentAsUrl.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Net.Http;
+
+namespace Microsoft.AspNetCore.TestHost
+{
+    /// <summary>
+    /// An implementation of <see cref="RequestContentOptions"/> that includes 
+    /// the [FromFornm] parameter as <see cref="IDictionary<>"/>
+    /// </summary>
+    public class IncludeContentAsUrl : RequestContentOptions
```

Comment:
I would like to change the name of this class. At least use **IncludeContentAsFormUrlEncoded** or something similar

Reply:
Fixed